client/views/popup.js

```diff
@@ -69,6 +69,11 @@ class Popup {
         this.el = document.createElement('div');
         this.el.className = 'discovery-view-popup';
 
+        Popup.els = Popup.els || new Set();
+        Popup.els.add(this.el);
+
+        Popup.elQueue = Popup.elQueue || new Set();
```

Comment:
Is there a reason to make it public? Maybe local variable instead?

Reply:
Maybe it does make sense. I'll change this.